faq.md

```diff
@@ -120,4 +182,4 @@ You can try replacing `event.fold(init) { ... }` with `Events.foldOne(event, ini
 
 ## Where are my exceptions?
 
-Per default, exceptions are not printed in Loci. So if your application just stops working, be sure to handle all possible exceptions yourself. For example, `.asLocal_?` might throw an exception if the Await times out. Therefore, it can help to wrap it in a `Try {}`.
\ No newline at end of file
+Per default, exceptions are not printed in Loci. So if your application just stops working, be sure to handle all possible exceptions yourself. For example, `.asLocal_?` might throw an exception if the Await times out. Therefore, it can help to wrap it in a `Try {}`.
```

Comment:
By now, exceptions should be logged (printed to the console by default); so this should not be an issue anymore